charts/testkube-api/templates/configmap.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
   {{- end }}
 data:
   executors.json: |-
-{{ .Files.Get "executors.json" | indent 4 }}
+{{ tpl (.Files.Get "executors.json") . | indent 4 }}
```

Comment:
If you are able to move it to the templates and define a template then please use `include` here like the other templates in the file.